deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -22,20 +24,20 @@ def __init__(self,
                n_graph_feat=128,
                **kwargs):
     """
-    Parameters
-    ----------
-    n_tasks: int
-      Number of tasks
-    n_atom_feat: int, optional
-      Number of features per atom.
-    n_pair_feat: int, optional
-      Number of features per pair of atoms.
-    n_hidden: int, optional
-      Number of units(convolution depths) in corresponding hidden layer
-    n_graph_feat: int, optional
-      Number of output features for each molecule(graph)
-
-    """
+        Parameters
+        ----------
+        n_tasks: int
+          Number of tasks
+        n_atom_feat: int, optional
+          Number of features per atom.
+        n_pair_feat: int, optional
+          Number of features per pair of atoms.
+        n_hidden: int, optional
+          Number of units(convolution depths) in corresponding hidden layer
+        n_graph_feat: int, optional
+          Number of output features for each molecule(graph)
+
+        """
```

Comment:
There's something a little weird going on here with the formatting. Did we change something in yapf?

(Don't need to address this in the current PR)